backend/src/main/java/com/constval/repook/service/ReportBookletCreationService.java

```diff
@@ -0,0 +1,18 @@
+package com.constval.repook.service;
+
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin(origins = "http://localhost:8081")
+@RestController
+@RequestMapping("/api")
+public class ReportBookletCreationService {
```

Comment:
Genau genommen wäre das ein Controller, in etwa `ReportBookletController` die Serviceschicht dient zu Operationen mit Repositories bzw als Zwischenschritt: Controller mit Restapi `ReportBookletController` -> ruft Service Methode auf aus `ReportBookletService` -> führt Repository oder ggf. andere Operation aus `ReportBookletRepository`.

Im Controller und Service haben wir dann verschiedene Apis & Methoden wie z.B. `delete` `save` `get` und würden dann nach Entitäten untergliedern, also `ReportBookletController` und `-Service` welchealle nötigen Operationen beinhalten

Reply:
Habe für die Änderung nicht so tief gedacht. ReportBookletCreationService sollte als Beispielklasse für "proof of concept" dienen für mich selber :'D
Aber danke für die Erläuterung :)